lua/telescope/pickers.lua

```diff
@@ -327,6 +327,7 @@ function Picker:find()
   assert(self.finder, "Finder is required to do picking")
 
   self.original_win_id = a.nvim_get_current_win()
+  state.set_global_key('original_buf_id', a.nvim_get_current_buf())
```

Comment:
We don't need a `state.set_global_key` just do that the api call when doing the get key access

Reply:
I am not sure what you mean

Reply:
Why do we need to to save it as state key? We just can get the current_buffer before doing edit. We are back there because the picker is already closed. Or am i wrong? He also already had that check in place it was just broken because of the normalize.

Reply:
That will not work because `get_current_buffer` before enter will get the telescope buffer, not the original one. Thats why we have to get the buffer *before* telescope has opened, hence setting a state key. Unless I am misunderstanding you

Reply:
Telescope should be closed at this point in time. That works on current master. Printing the `bufnr` before edit gives me the correct one

Reply:
You are right that telescope should be closed. My bad